Multivariate chapters update

patmanas opened the pull request:
add Robert in credits

patmanas committed:
update TOC: remove non Gaussian

patmanas committed:
add covariance, correl and multi gaussian

patmanas committed:
update toc to include changes

patmanas committed:
update overview

patmanas committed:
couple of typos in multi variables

patmanas committed:
typos

patmanas committed:
update route to images in correl and cov

patmanas committed:
typo in toc

patmanas committed:
update route to figs gaussian

patmanas committed:
remove others parts from TOC

Tom-van-Woudenberg committed:
Merge branch '2025-draft' into pmn_multi

Tom-van-Woudenberg committed:
Update credits.md

Because figure was removed from overview page

Tom-van-Woudenberg committed:
Add credits

Tom-van-Woudenberg committed:
converted iframes to iframe figures

patmanas committed:
Merge branch '2025-draft' into pmn_multi

patmanas committed:
small edit in credits

Tom-van-Woudenberg commented:
@patmanas , did you really want to close this?

patmanas commented:
@Tom-van-Woudenberg , I was not sure if you wanted to have it already merged there or wait for later so that is why I closed it. Shall I merge it?

Tom-van-Woudenberg commented:
We can merge it already, it's not shown in the 2025 version because those chapter are kept out in the toc. Would it be useful to have a review from someone who knows the topic better than me?

Tom-van-Woudenberg commented:
And could you maybe describe a bit what you've changed? :)

patmanas commented:
Sure.
Since the order of the topics has changed and observation theory and propagation of uncertainty goes at the end, there was a need to introduce some topics that were previously seen during those weeks:
- Added a thorough description of covariance and correlation, including geometric interpretation and exercises;
- Updated the multivariate Gaussian distribution page and corrected some typos;
- Removed the last part of practical applications as it was an intro to reliability and function of random variables that was too much with the extra material added;
- Removed non-Gaussian distributions, as it was a very brief intro meant to justify the use of a copula in the project. This year the project will not contain a copula, so it was not needed. It is content that will be covered in latter courses.

Tom-van-Woudenberg commented:
> Would it be useful to have a review from someone who knows the topic better than me?

@patmanas

fmeer commented:
I could review the chapter, but after week 1.1. @jialeidng can also not do it any sooner I suppose

patmanas commented:
@Tom-van-Woudenberg, I asked @MaxRamgraber to have a look if he can.

@LotfiMassarweh, could you have a look? You will be using this part later on in uncertainty propagation.

I have added them as reviewers.

Tom-van-Woudenberg committed:
Merge branch '2025-draft' into pmn_multi

Tom-van-Woudenberg committed:
Update Correlation_and_Covariance.md

fmeer committed:
Spell check

fmeer committed:
mathrm for Cov

fmeer committed:
Cov and matrix formatting

fmeer committed:
Smoothen transition from rho to Cov

fmeer commented:
Reviewed two versions of this chapter today can confirm this one is better than the previous one ;) 

I made some cosmetic changes, and expanded a little bit the first equations of the Bivariate page, to make it easier to follow for who is not entirely familiar with definitions from the Correlations&Covariance page. @patmanas would you support what I did to the second equation on Bivariate page?

patmanas commented:
Hi @fmeer, fine with me. Thanks for the review!

fmeer committed:
Merge branch '2025-draft' into pmn_multi
